base/src/main/java/com/smartdevicelink/proxy/rpc/enums/AppInterfaceUnregisteredReason.java

```diff
@@ -119,6 +119,15 @@
 	 * @since SmartDeviceLink 4.1
 	 */
 	UNSUPPORTED_HMI_RESOURCE,
+
+
+	/**
+	 * By getting this value, SDL should unregister the application to allow the HMI to close the application.

```

Comment:
As per the proposal, this description is part of the HMI_API. The description for the app should be something like:

"The application is unregistered due to hardware resource constraints. The system will shortly close the application to free up hardware resources." This text should be part of the rpc_spec and align with generated code.